src/Printer.java

```diff
@@ -0,0 +1,18 @@
+import java.io.PrintStream;
+
+public class Printer {
```

Comment:
This one might be a good place to start on our next session.

I made a comment in Main about it knowing things maybe Printer should know.
Judging by this class you may have felt Printer wouldn't be very reusable if it knew more.

There is lots of good grey area to talk about design at this boundary, and our outcome might inform you choices at the other end for getting input too!
 
ie: currently Printer just prints things, but it still needs the help of others to ensure what it receives is ready to be printed as-is. Who should care about this problem?

Reply:
If we push all the ugly stuff to the corners of our system, maybe it's the Printer?